includes/definitions/siae-alfoplus2.yaml

```diff
@@ -0,0 +1,12 @@
+os: alfoplus2

```

Comment:
Does this device use the same MIBs or different MIBs as other saie devices?  It may share an OS and you probably don't want this specific of an OS name if that is a case.

If MIBs are only for this device class, this name is ok.

Reply:
The MIBs came directly from SIAE in a zip called:  ALFOplus2_MIB_R1.5.0.  However, it's likely several of the MIB sub-files are shared with their other devices.  

They don't seem to have an OS name that I could find.  Should I just rename it to "siae"?  I already named the mibdir and icons "siae".

Reply:
Looks like they do have an OS name for this device.  SM-OS.  I'll make the changes.